app/CreateDeal.tsx

```diff
@@ -61,6 +60,12 @@ export function CreateDeal(props: {
                 />
               </div>
             </FormGroup>
+            <FormGroup>
+              <FormLabel>Unit</FormLabel>
+              <FormCheck type="radio" label="kg"></FormCheck>
+              <FormCheck type="radio" label="kg"></FormCheck>
+              <FormCheck type="radio" label="kg"></FormCheck>
+            </FormGroup>
```

Comment:
this isn't really part of the type guard, and also doesn't make sense as standalone code. Gotta look through the PR before putting it up for review

Reply:
https://github.com/auberginearms/groceryDatabaseApp/pull/28/commits/658cabc7ab02e178cc5b342397cd903babab3116